src/test/java/cz/fi/muni/pa165/Model/Entity/CarTest.java

```diff
@@ -44,46 +48,30 @@ public void testChangeState() throws Exception {
 
         car.changeState(CarState.DISCARDED);
         assertEquals(car.getCarState(), CarState.DISCARDED);
-    }
-
-    @Test
-    public void testSetSerialNumber() throws Exception {
-        Car car = create();
-
-        car.setSerialNumber("C3PO");
-        assertEquals(car.getSerialNumber(), "C3PO");
-    }
-
-    @Test
-    public void testSetRegPlateNumber() throws Exception {
-        Car car = create();
-
-        car.setRegPlateNumber("000598");
-        assertEquals(car.getRegPlateNumber(), "000598");
-    }
-
-    @Test
-    public void testSetManufacturer() throws Exception {
-        Car car = create();
 
-        car.setManufacturer("Škoda Auto");
-        assertEquals(car.getManufacturer(), "Škoda Auto");
+        car.changeState(CarState.DISCARDED);
+        assertEquals(car.getCarState(), CarState.DISCARDED);
     }
 
     @Test
-    public void testSetType() throws Exception {
+    public void testChangeStateInvalidTransitions() throws Exception {
         Car car = create();
 
-        car.setType("top kek");
-        assertEquals(car.getType(), "top kek");
-    }
-
-    @Test
-    public void testSetNumberOfSeats() throws Exception {
-        Car car = create();
+        car.changeState(CarState.DISCARDED);
+        assertEquals(car.getCarState(), CarState.DISCARDED);
 
-        car.setNumberOfSeats(3);
-        assertEquals(car.getNumberOfSeats(), 3);
+        try {
+            car.changeState(CarState.OK);
```

Comment:
I am unable to find a good documentation with an example on how to exactly use `assertThrows()`, can you find me any?